services/reviewService.js

```diff
@@ -0,0 +1,8 @@
+const reviewDao = require('../models/reviewDao')
+
+const findReview = async (productId) => {
```

Comment:
```suggestion
const findReviewByProductId = async (productId) => {
```
- 함수 이름 고치실때 한 파일에서만 단편적으로 수정하시기 보다는 연관되어 있는 모든 함수의 이름을 한 번씩 체크하셔서 같이 고치시는 것이 좋습니다 :)